seq/src/cli.rs

```diff
@@ -0,0 +1,30 @@
+use clap::{
+    crate_authors, crate_description, crate_name, crate_version, App, AppSettings::ColoredHelp, Arg,
+};
+
+pub(crate) fn create_app<'a, 'b>() -> App<'a, 'b> {
+    App::new(crate_name!())
+        .version(crate_version!())
+        .author(crate_authors!())
+        .about(crate_description!())
+        .help_message("Display help information.")
+        .version_message("Display version information.")
+        .help_short("?")
+        .settings(&[ColoredHelp])
+        .arg(Arg::with_name("FIRST INCREMENT LAST").multiple(true))
+        .arg(
+            Arg::with_name("SEPERATOR")
+                .short("s")
+                .long("separator")
+                .help("use STRING to separate numbers (default: \\n)")
```

Comment:
I think the default text would be better with square bracket to be similar to the normal default text that clap uses